recipes/hyperopt/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "hyperopt" %}
+{% set version = "0.1.1a" %}
+# 0.1.1a is not a real version; but 0.1.0 was released a year ago, so this will do for now.
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hyperopt/hyperopt.git
```

Comment:
Please don't build from urls https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos
If a new version hasn't been released please request/make a release (it seems that they are interested in some help in that regard https://github.com/hyperopt/hyperopt/issues/237#issuecomment-139573968)

Reply:
@CJ-Wright The last comment in that issue was 2.5 years ago... 

Alright, I'll try, I'll try...